### DISCUSSION TITLE
Propose donations

### DISCUSSION THREAD

**Discussion by liZe** [Kozea/Radicale#1382] [General]
I've received (3, 4?) mails during the last weeks asking me how donations could be done to Radicale. I'm not personally interested in these donations, but it can be great for main contributors to buy coffee/beer/champagne thanks to these donations.

@Unrud what do you think of that?

**Comment by pbiering** [comment]
@gentooboontoo : what do you think about?

**Comment by gentooboontoo** [comment]
As a GitHub-hosted project, enabling «Github Sponsors» would be the more efficient approach for accepting donations. It could potentially serve as an incentive for consistent contributors. I will look at it.
